sdk/cosmosdb/cosmos/package.json

```diff
@@ -11,98 +13,122 @@
   },
   "files": [
-    "dist",
-    "dist-esm"
+    "changelog.md",
+    "dist/",
+    "dist-esm/",
+    "LICENSE",
+    "src/",
+    "tsconfig.json"
   ],
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/cosmosdb/cosmos#readme",
   "types": "./dist/index.d.ts",
   "engine": {
-    "node": ">=6.0.0"
+    "node": ">=8.0.0"
   },
   "scripts": {
-    "clean": "rimraf dist && rimraf dist-esm",
-    "lint": "tslint --project ./src/tsconfig.json",
-    "format": "prettier --write --config .prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\"",
-    "check-format": "prettier --list-different --config .prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\"",
-    "compile": "echo Using TypeScript && tsc --version && tsc -b --pretty",
-    "compile:samples": "tsc -b samples",
-    "docs": "typedoc --tsconfig ./src/tsconfig.json --excludePrivate --mode file --out ./dist/docs ./src",
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:samples": "tsc -b samples",
+    "build:src": "echo Using TypeScript && tsc --version && tsc -b --pretty",
+    "build:test": "tsc -b src test --verbose",
+    "build": "npm run clean && npm run extract-api && npm run bundle && node writeSDKVersion.js",
     "bundle": "rollup -c",
     "bundle-types": "node bundle-types.js",
-    "build": "npm run clean && npm run check-format && npm run lint && npm run compile && node writeSDKVersion.js && npm run docs && npm run bundle && npm run bundle-types",
-    "test": "tsc -b src test --verbose && mocha --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=- -r esm -r dotenv/config -r ./test/common/setup.js \"./test/**/*.spec.js\" --timeout 100000",
+    "check-format": "prettier --list-different --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-esm temp types *.tgz *.html *.log *.tsbuildinfo test/**/*.{js,js.map,d.ts}",
+    "docs": "typedoc --tsconfig ./src/tsconfig.json --excludePrivate --mode file --out ./dist/docs ./src",
+    "extract-api": "npm run build:src && api-extractor run --local",
+    "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "mocha -r esm -r dotenv/config -r ./test/common/setup.js \"./test/**/*.spec.js\" --timeout 100000",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint \"src/**/*.ts\" \"test/**/*.ts\" -c ../../.eslintrc.json --fix --fix-type [problem,suggestion]",
+    "lint": "eslint -c ../../.eslintrc.json src test --ext .ts -f html -o cosmos-lintReport.html || exit 0",
+    "pack": "npm pack 2>&1",
+    "prebuild": "npm run clean",
     "prepack": "npm install && npm run build",
```

Comment:
Please remove the `prepack` line, in particular we should **never** run `npm install` in any of our scripts

Reply:
Removed